integration-tests/src/tests/client/cold_storage.rs

```diff
@@ -224,5 +225,6 @@ fn test_cold_db_head_update() {
 
         assert_eq!(head, &cold_head_in_cold);
         assert_eq!(head, &cold_head_in_hot);
+        assert_eq!(h, test_get_cold_head_height() as u64);
```

Comment:
Testing metrics can be flaky, because tests are run in parallel and can update metrics in parallel.

Reply:
Ok, removing it, don't need it anyway.